### PR TITLE
[docs] Build notebooks from Markdown

### PR DESCRIPTION
**Pull request by stevhliu**
This PR adds functionality for building a Colab notebook from Markdown :)

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Shouldn't this already show here: https://github.com/huggingface/notebooks/tree/main?

**Comment by stevhliu** [comment]
Oops, I've set the notebooks to be built in the existing `diffusers` folder at https://github.com/huggingface/notebooks/tree/main

**Comment by patrickvonplaten** [comment]
I'm still not seeing the automatically created notebooks. I think `diffusers-doc` is a good notebook name actually, so would be good to build it there. It's just that our doc-builder does not yet seem to work for diffusers no? Note how it builds docs for transformers: https://github.com/huggingface/notebooks/commits?author=HuggingFaceDocBuilder

**Comment by stevhliu** [comment]
I think the notebooks are automatically created after merging this, and the build documentation workflow gets run (not sure if it happens in PR previews). @sgugger do you know if there is anything else we need to do here?

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
     with:
       commit_sha: ${{ github.sha }}
       package: diffusers
+      notebook_folder: diffusers
```
Comment (sgugger) [inline .github/workflows/build_documentation.yml:16]:
```suggestion
      notebook_folder: diffusers_doc
```
To be consistent with Transformers

**Review by sgugger — COMMENTED** [review]
Yes, the notebooks will only be built in the main build documentation job after this PR is merged.

**Comment by patrickvonplaten** [comment]
Awesome, thanks @sgugger - let's give it a try then!

**Pull request merged**